examples/vpc/versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.9.0"
+    }
+    awsutils = {
```

Comment:
@denissimonovski we can remove this (and the provider and other usage) now, as of this [PR](https://github.com/cloudposse/terraform-aws-s3-bucket/pull/158). Sorry for the run-around, but we may as well fix this now.